src/ckeditor_markdown.js

```diff
@@ -0,0 +1,62 @@
+import JnClassicEditor from "./base/jn-classic-editor";
+import Bold from "@ckeditor/ckeditor5-basic-styles/src/bold.js";
+import Essentials from "@ckeditor/ckeditor5-essentials/src/essentials.js";
+import Heading from "@ckeditor/ckeditor5-heading/src/heading.js";
+import List from "@ckeditor/ckeditor5-list/src/list.js";
+import ListProperties from "@ckeditor/ckeditor5-list/src/listproperties.js";
+import Paragraph from "@ckeditor/ckeditor5-paragraph/src/paragraph.js";
+import { Link } from "@ckeditor/ckeditor5-link";
+import { Markdown } from "@ckeditor/ckeditor5-markdown-gfm";
+import SourceEditing from "@ckeditor/ckeditor5-source-editing/src/sourceediting";
+
+class Editor extends JnClassicEditor {}
+
+// Plugins to include in the build.
+Editor.builtinPlugins = [
```

Comment:
Er dette en fullstendig liste over plugins som vi vil støtte?

Trenger vi å ha flere? For eksempel fra `/basic-styles/`, `Image `og etc.